ADMVEDU95/ADMVEDU95/Presentation/Category/Presenter/CategoryPresenter.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CategoryPresenter.swift
+//  ADMVEDU95
+//
+//  Created by Satishur, Oleg on 27.04.2021.
+//
+
+import Foundation
+
+protocol CategoryViewPresenterProtocol: class {
```

Comment:
CategoryViewProtocol and CategoryPresenterProtocol but please don't mix View and Presenter in protocols name